tasks/configure.yml

```diff
@@ -215,15 +215,17 @@
     mode: u=rwX,g=rwXs,o=rx
   notify: restart postgresql
 
-- name: PostgreSQL | Add to PATH (/etc/profile.d/postgres.sh)
-  template:
-    src: "etc_profile_d_postgres.sh.j2"
-    dest: "/etc/profile.d/postgres.sh"
-    mode: 0644
-  when: postgresql_add_to_path and ansible_os_family == "RedHat"
```

Comment:
@timothylytle Have you tested a setup of MiaRec on Centos after removing this piece?

I past, I had to add this code to deal with issues when installing python packages.
At that time, some of packages were compiled from source code and required `pkgconfig` binary.
It may be not necessary now because we normally try to install the pre-built (wheel) packages with pip.

Reply:
Also, if we should remove `postgresql_add_to_path` variable from `ansible-miarec/vars/db.yml`

Reply:
I am closing this comment. The variable `postgresql_add_to_path` is not required now.

Reply:
I was able to deploy successfully on centos 7 after this change,      Also I will remove the variable `postgresql_add_to_path` from `ansible-miarec/vars/db.yml`

Reply:
I have merged this PR and deleted the variable form the parent project.